website/addons/osfstorage/model.py

```diff
@@ -28,17 +30,15 @@ class OsfStorageNodeSettings(StorageAddonBase, AddonNodeSettingsBase):
     complete = True
     has_auth = True
 
-    root_node = fields.ForeignField('OsfStorageFileNode')
-    file_tree = fields.ForeignField('OsfStorageFileTree')
-
-    # Temporary field to mark that a record has been migrated by the
-    # migrate_from_oldels scripts
-    _migrated_from_old_models = fields.BooleanField(default=False)
+    root_node = fields.ForeignField('StoredFileNode')
 
     @property
     def folder_name(self):
         return self.root_node.name
 
+    def get_root(self):
```

Comment:
There's not another option that seems great to me, developers need a way to get the wrapped variant of the root node and I'd prefer not to migrate what doesn't have to be migrated.

It doesn't feel super fragile to me; if you don't call `get_root()` nothing will work for you. :smiling_imp: